SofaKernel/modules/Sofa.Config/CMakeLists.txt

```
@@ -60,7 +60,9 @@ option(SOFA_WITH_OPENGL "Compile Sofa with OpenGL support. This will define the
 option(SOFA_NO_UPDATE_BBOX "Compile Sofa with the SOFA_NO_UPDATE_BBOX macro defined." OFF)
 option(SOFA_DUMP_VISITOR_INFO "Compile Sofa with the SOFA_DUMP_VISITOR_INFO macro defined." OFF)
 if(MSVC)
-    option(SOFA_VECTORIZE "Enable the use of SSE2 instructions by the compiler (Only available for MSVC)." OFF)
+    option(SOFA_VECTORIZE "(deprecated) Enable the use of SSE2 instructions by the compiler (Only available for MSVC)." OFF)
+    option(SOFA_ENABLE_SIMD "Enable the use of SIMD instructions by the compiler (AVX/AVX2 for msvc, march=native for gcc/clang)." OFF)
```

```suggestion
    option(SOFA_ENABLE_SIMD "Enable the use of SIMD instructions by the compiler (AVX/AVX2 for msvc)." OFF)
```
`SOFA_ENABLE_SIMD` is available for MSVC only, right?

---

I have the same for linux/gcc/clang in an other branch (and mac/clang as well)